data/engines.txt

```diff
@@ -223,7 +223,7 @@ outfit "X5200 Ion Steering"
 	"turn" 2174
 	"turning energy" 3.5
 	"turning heat" 7.3
-	description "The X5200 is a steering system so powerful that only a handful of ships use it. It is standard equipment for Republic cruisers and carriers, ships so large that they are practically a self-contained city."
+	description "The X5200 is a steering system so powerful and large that only a handful of ships can install it. It is useful for capital ships so large they are practically self-contained cities.
```

Comment:
1st ya forgot the " at the end

2nd, I did an attempt to improve the description: "The X5200 is a steering system so large and powerful that only a handful of ships can install it. This means that this steering system is rarely sold due to its limited market."

I entirely rewrote the second sentence since both sentences basically talked about how only giant ships can use one